roles/rhcos-post-install/templates/infra-node-machineset.yml.j2

```diff
@@ -54,7 +54,7 @@ items:
             - filters:
               - name: tag:Name
                 values:
-                - {{rhcos_cluster_name.stdout}}_worker_sg
+                - {{rhcos_cluster_name.stdout}}-worker-sg
```

Comment:
In the last cluser I deployed I still see the prior format

Ex:
```
            securityGroups:
            - filters:
              - name: tag:Name
                values:
                - akrzosrhperfocpauto0-st6dn_worker_sg
```

This was with build 2019-02-24-045124 so maybe you ran a newer build to see this change?

Reply:
I saw the change when using the same release. The pbench and infra nodes are using the default security group as it's not able to find the security group created for workers. About the output you pasted, is it from the machinesets we create or the worker node?

Reply:
Yep your right, I accidentally grabbed the snippet from the infra nodes when I dumped the machineset yaml... so disreguard my comment here.